app/models/background.rb

```diff
@@ -7,4 +7,37 @@ class Background < ApplicationRecord
   def validate_size
     errors.add(:base, "Need to select at least 2 pipeline runs.") if pipeline_outputs.size < 2
   end
+
+  def summarize
```

Comment:
again. i think it's better to do this through database calculation. This would be slow for a direct query.

Reply:
how does one run a SQL query from this magical place?

Reply:
check out for example the latest version of app/models/background.rb!